### ISSUE TITLE
Can't run even compile generated project

### ISSUE THREAD

**Issue by Gargo** [Circular-Studios/Dash-CLI#8]
```
Building game...
Error(s) compiling project:
Error: module exception is in file 'C:\Users\╨Р╨┤╨╝╨╕╨╜╨╕╤Б╤В╤А╨░╤В╨╛╤А\AppData\Roaming\dub\packages\derelict-util-2.1.0\derelict-util\source\derelict\util\exception.d' which cannot be read
import path[0] = C:\Users\╨Р╨┤╨╝╨╕╨╜╨╕╤Б╤В╤А╨░╤В╨╛╤А\AppData\Roaming\dub\packages\derelict-util-2.1.0\derelict-util\source
import path[1] = E:\D\dmd2\windows\bin\..\..\src\phobos
import path[2] = E:\D\dmd2\windows\bin\..\..\src\druntime\import
dmd failed with exit code 1.
```

**Comment by brooklynlittell** [comment]
Could you verify the file `C:\Users\╨Р╨┤╨╝╨╕╨╜╨╕╤Б╤В╤А╨░╤В╨╛╤А\AppData\Roaming\dub\packages\derelict-util-2.1.0\derelict-util\source\derelict\util\exception.d` exists?

And try running `dub upgrade` to make sure the dependencies are loaded.

**Comment by Gargo** [comment]
@BrandonLittell 
dub is installed under windows via executable and there is no such command. Could it be the problem?

**Comment by brooklynlittell** [comment]
Ah, yeah, try adding dub.exe to your PATH

**Comment by Gargo** [comment]
@BrandonLittell 
could you explain how to do that?

**Comment by brooklynlittell** [comment]
In the start menu, search for "environment" and select "Edit the system environment variables." A `System Properties` dialog will open, select the "Environment Variables" button.

Find "Path" in "System Variables" and click edit, and add the location of your dub.exe. Restart your machine and dub should be all setup.

**Comment by Gargo** [comment]
@BrandonLittell 
No, still have the same issue. Maybe if it is because I didn't install dub in default location and choose it by myself?
